extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/TrackedMeasurementsContext.tsx

```diff
@@ -81,6 +86,29 @@ function TrackedMeasurementsContextProvider(
         },
       });
     },
+
+    jumpToSameImageInActiveViewport: (ctx, evt) => {
```

Comment:
Will this be the same logic if there is only one measurement? What I'm asking is, is `jumpToSameImage = jumpToFirstMeasurement` if there is only one measurement?

Reply:
It doesn’t matter how many measurements there are, it will still be the same slice the user was on